extensions/google-java/src/main/kotlin/build/buf/connect/extensions/JavaErrorParser.kt

```diff
@@ -12,12 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package build.buf.connect.extensions
+package com.connectrpc.extensions
 
-import build.buf.connect.AnyError
-import build.buf.connect.ConnectErrorDetail
-import build.buf.connect.ErrorDetailParser
 import build.buf.google.rpc.Status
```

Comment:
We should update the `buf.gen.yaml` as well to put the generated code in `com.connectrpc.*` too.